apps/ledger-live-mobile/docs/llm_e2e_testing.md

```diff
@@ -0,0 +1,200 @@
+pnpm  --filter="live-mobile..." --filter="ledger-live" --no-frozen-lockfile --unsafe-perm pod &&
+pnpm build:llm:deps && 
+pnpm mobile detox clean-framework-cache && pnpm mobile detox build-framework-cache &&
+pnpm mobile e2e:build -c android.emu.debug && 
+pnpm mobile e2e:build -c ios.sim.debug
+```
+
+---
+---
+
+## Local Environment Setup
+
+Writing and running Detox tests requires Xcode for iOS and Android Studio (along with the SDK and emulator tools) for Android. Setting up your local environment for running the tests can be tricky, as there are two different platforms with their own intricacies as well as the working with React Native. The best place to setup both Android and iOS is to start with [React Native's own documentation](https://reactnative.dev/docs/environment-setup), however see the sections below for each platform to deal with specific quirks.
+
+Detox also has good documentation for [Android](https://wix.github.io/Detox/docs/introduction/android-dev-env/) and [iOS](https://wix.github.io/Detox/docs/introduction/ios-dev-env) environment setup. Both guides differ slightly from the React Native ones but they may help to overcome some issues you have.
+
+### Additional Android setup steps
+
+We've found that it's better to install Java manually rather than with Homebrew, as they are often installed in locations on your computer that Detox and Android Studio aren't expecting. Follow [this](https://techoral.com/blog/java/install-openjdk-11-on-mac.html) guide.
+
+### Additional iOS setup steps
+
+Make sure to install Wix's `applesimutils` (info found [here](https://wix.github.io/Detox/docs/introduction/ios-dev-env)).
+
+---
+---
+
+## Test Setup and Execution
+
+Clean your local environment (the main thing this step does is remove `node_modules` and previous iOS and Android mobile app builds):
+
+```bash
+pnpm clean
+```
+
+Install dependencies:
+
+```bash
+pnpm i
+```
+
+> There is a filtered version of this command which should be quicker and includes only dependencies needed for LLM: `pnpm i --filter="live-mobile..." --filter="ledger-live"`.
+
+Build dependencies for the mobile app:
+
+```bash
+pnpm build:llm:deps
+```
+
+### Android
+
+Verify you have an emulator [installed](https://developer.android.com/studio/run/managing-avds) and have that match the Detox `avdName` (currently 'Nexus_6') in the `detox.config.js` file. Be sure to make the device the correct architecture and system image (currently x86_64 if you are on an Intel mac and arm64_v8a if you are on an M1).
+
+- Build the apps
+  - Debug: `pnpm mobile e2e:build -c android.emu.debug`
+  - Release: `pnpm mobile e2e:build -c android.emu.release`
+- Run the tests
+  - Debug: First, run `pnpm mobile start` to run Metro bundler, then in a separate terminal window run `pnpm mobile e2e:test -c android.emu.debug`
+  - Release: `pnpm mobile e2e:test -c android.emu.release`
+
+### iOS
+
+Use the exact same commands as above Android ones but replace `android.emu` with `ios.sim`.
```

Comment:
Please copy the commands with the correct commands

Reply:
Done